test/DifferentialExpressionMatrixSet_basic_test.py

```diff
@@ -128,37 +130,39 @@ def test_save_dem_set_mismatched_genomes(self):
                 "label": "not_so_odd"
             }]
         }
-        with self.assertRaises(ValueError) as err:
+        with pytest.raises(
+            ValueError,
+            match="All Differential Expression Matrix objects in the set must use the same genome reference."
+        ):
```

Comment:
I had to correct the error message here; previously, the assertion statement was not run as it was incorrectly indented and followed the `self.getImpl().save_differential_expression_matrix_set_v1(...)`, which throws an error and stops execution. During the search and replace odyssey, I corrected the indentation and saw that the test failed. I looked up the relevant code in the implementation file and put in the correct error message, and as if by magic, the test passed.